### ISSUE TITLE
Missing prompts

### ISSUE THREAD

**Issue by Muennighoff** [ExpressAI/reStructured-Pretraining#2]
Nice work! I think some of the prompts from your appendix are missing in DataLab?

Looking in [this directory](https://github.com/ExpressAI/DataLab/tree/main/datalabs/operations/prompt), I couldn't find e.g. prompts for sentence expansion like
```
source: TEXT: {summary} QUERY: What details would you include in a storyline to make it more
engaging and informative?
```

Is there a way to get all prompts you used similar to having all datasets in `https://github.com/ExpressAI/DataLab/blob/main/datasets/rst/rst.py`, so that one can easily reconstruct your full dataset for training? Thanks! 🤗

**Comment by Muennighoff** [comment]
Also is `daily_mail_category` supposed to work with the topic classification prompts? It sounds like topic clf to me, but doesn't seem to work 🧐

When running:
```
ds = load_dataset("rst", "daily_mail_category")
res = ds['train'].apply(template_tc2)
print(next(res))
```
I get
```
TypeError                                 Traceback (most recent call last)
[<ipython-input-39-3d757e349047>](https://localhost:8080/#) in <module>
      3 ds = load_dataset("rst", "daily_mail_category")
      4 res = ds['train'].apply(template_tc2)
----> 5 print(next(res))

[/usr/local/lib/python3.8/dist-packages/datalabs/arrow_dataset.py](https://localhost:8080/#) in apply_basic(self, func, prefix, num_proc)
    909         ]:
    910             for sample in self.__iter__():
--> 911                 labels = self._info.task_templates[0].labels
    912                 labels_to_answers = dict(zip(range(len(labels)), labels))
    913                 yield func(sample, labels_to_answers)

TypeError: 'NoneType' object is not subscriptable
```

cc @yyy-Apple @pfliu-nlp

**Comment by yyy-Apple** [comment]
> Nice work! I think some of the prompts from your appendix are missing in DataLab?
> 
> Looking in [this directory](https://github.com/ExpressAI/DataLab/tree/main/datalabs/operations/prompt), I couldn't find e.g. prompts for sentence expansion like
> 
> ```
> source: TEXT: {summary} QUERY: What details would you include in a storyline to make it more
> engaging and informative?
> ```
> 
> Is there a way to get all prompts you used similar to having all datasets in `https://github.com/ExpressAI/DataLab/blob/main/datasets/rst/rst.py`, so that one can easily reconstruct your full dataset for training? Thanks! 🤗

Yes, we have not added those prompts to DataLab yet, but the prompts we used for training are listed in full in the Appendix of the paper.

**Comment by yyy-Apple** [comment]
> Also is `daily_mail_category` supposed to work with the topic classification prompts? It sounds like topic clf to me, but doesn't seem to work 🧐
> 
> When running:
> 
> ```
> ds = load_dataset("rst", "daily_mail_category")
> res = ds['train'].apply(template_tc2)
> print(next(res))
> ```
> 
> I get
> 
> ```
> TypeError                                 Traceback (most recent call last)
> [<ipython-input-39-3d757e349047>](https://localhost:8080/#) in <module>
>       3 ds = load_dataset("rst", "daily_mail_category")
>       4 res = ds['train'].apply(template_tc2)
> ----> 5 print(next(res))
> 
> [/usr/local/lib/python3.8/dist-packages/datalabs/arrow_dataset.py](https://localhost:8080/#) in apply_basic(self, func, prefix, num_proc)
>     909         ]:
>     910             for sample in self.__iter__():
> --> 911                 labels = self._info.task_templates[0].labels
>     912                 labels_to_answers = dict(zip(range(len(labels)), labels))
>     913                 yield func(sample, labels_to_answers)
> 
> TypeError: 'NoneType' object is not subscriptable
> ```
> 
> cc @yyy-Apple @pfliu-nlp

Currently, the RST datasets are not aligned with DataLab prompting functionality, we do not recommend using the `apply(template)` function.

**Comment by Muennighoff** [comment]
> Currently, the RST datasets are not aligned with DataLab prompting functionality, we do not recommend using the `apply(template)` function.

So is it currently possible to recreate your dataset for training?
